tests/test_dataframe_mapper.py

```diff
@@ -913,28 +882,6 @@ def test_with_car_dataframe(cars_dataframe):
     assert scores.mean() > 0.30
 
 
-@pytest.mark.skipIf(parse_version(sklearn_version) < parse_version('0.16'))
-def test_direct_cross_validation(iris_dataframe):
```

Comment:
Why are we removing this test? It should work for sklearn > 0.16.

Reply:
I was having trouble with skipIf -- seems like it got removed.. I was just trying to get circleci working on my fork . Let me spend some more time to figure out what's happening over here

Reply:
I included the test back. Removed parse_version this as it's not relevant anymore and also deleted custom cross_val_score as the new scikit version support DataFrame.